Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Thsi dockerfile builds a docker image for the example site.
+
+# Use an official Python runtime as a parent image
+FROM python:3.8
+LABEL maintainer="hello@wagtail.io"
+
+# Set environment varibles
+ENV PYTHONUNBUFFERED 1
+ENV DJANGO_ENV dev
+
+# we're only going to copy the requirements for pip first, so if we're working
+# on the example site and testing docker builds we don't have to rebuid the
+# "pip requirements layers" every time we change a file.
+
+# copy local pip dependences
+COPY ./grapple /code/grapple
+COPY ./setup.cfg /code/setup.cfg
+COPY ./setup.py ./code/setup.py
+COPY ./requirements-channels.txt /code/requirements-channels.txt
+COPY ./requirements.txt /code/requirements.txt
+
+# copy example site requires file
+COPY ./example/requirements.txt /code/example/requirements.txt
+COPY ./example/requirements-channels.txt /code/example/requirements-channels.txt
```

Comment:
ditto

Reply:
If you wanna make that change go for it.  It isn't functionally different and makes it hard to see which file was added or removed on diffs, and also creates a single layer for all files so changing one invalidates layers for all.